### PR TITLE
Renamed the Agent classes and added Agent at the End.

### PR DESCRIPTION
**Pull request by azimme2s**
Also added some new Agent Skeleton classes

Let me know what you guys think and if I forgot some classes

### CODE REVIEW

File: src/main/java/org/awesomebakery/agents/OvenAgent.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    }
+
+    @Override
+    public String getServiceType() {
+        return SERVICE_TYPE;
+    }
+
+    @Override
+    public void onMessageReceived(ACLMessage message) {
+        // TODO Auto-generated method stub
```
Comment (sonarcube) [inline src/main/java/org/awesomebakery/agents/OvenAgent.java]: INFO Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)

File: src/main/java/org/awesomebakery/agents/BoxingAgent.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    }
+
+    @Override
+    public String getServiceType() {
+        return SERVICE_TYPE;
+    }
+
+    @Override
+    public void onMessageReceived(ACLMessage message) {
+        // TODO Auto-generated method stub
```
Comment (sonarcube) [inline src/main/java/org/awesomebakery/agents/BoxingAgent.java]: INFO Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)

File: src/main/java/org/awesomebakery/agents/KneadingAgent.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    }
+
+    @Override
+    public String getServiceType() {
+        return SERVICE_TYPE;
+    }
+
+    @Override
+    public void onMessageReceived(ACLMessage message) {
+        // TODO Auto-generated method stub
```
Comment (sonarcube) [inline src/main/java/org/awesomebakery/agents/KneadingAgent.java]: INFO Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)

**Comment by sonarcube** [comment]
SonarQube analysis reported 6 issues
* ![INFO][INFO] 6 info

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [DoughPreparerAgent.java#L21](https://github.com/HBRS-MAAS/project-awesomebakery/blob/1dc4b5ddd71b5d3af8bc3f2b28cf407d951bf0d4/src/main/java/org/awesomebakery/agents/DoughPreparerAgent.java#L21): Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)
1. ![INFO][INFO] [DoughResterAgent.java#L21](https://github.com/HBRS-MAAS/project-awesomebakery/blob/1dc4b5ddd71b5d3af8bc3f2b28cf407d951bf0d4/src/main/java/org/awesomebakery/agents/DoughResterAgent.java#L21): Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)
1. ![INFO][INFO] [TruckAgent.java#L21](https://github.com/HBRS-MAAS/project-awesomebakery/blob/1dc4b5ddd71b5d3af8bc3f2b28cf407d951bf0d4/src/main/java/org/awesomebakery/agents/TruckAgent.java#L21): Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

File: src/main/java/org/awesomebakery/agents/DoughResterAgent.java [hunk 4]
```diff
@@ -2,12 +2,12 @@
 
 import jade.lang.acl.ACLMessage;
 
-public class DoughRester extends ServiceAgent {
+public class DoughResterAgent extends ServiceAgent {
 	private static final long serialVersionUID = 3213192460512915864L;
 
-	public static final String SERVICE_TYPE = "DoughRester";
+	public static final String SERVICE_TYPE = "DoughResterAgent";
```
Comment (mwiktorin) [inline src/main/java/org/awesomebakery/agents/DoughResterAgent.java]: I would keep the old name here

File: src/main/java/org/awesomebakery/agents/DoughPreparerAgent.java [hunk 5]
```diff
@@ -2,12 +2,12 @@
 
 import jade.lang.acl.ACLMessage;
 
-public class DoughPreparer extends ServiceAgent {
+public class DoughPreparerAgent extends ServiceAgent {
 	private static final long serialVersionUID = -7194206594373177352L;
 
-	public static final String SERVICE_TYPE = "DoughPreparer";
+	public static final String SERVICE_TYPE = "DoughPreparerAgent";
```
Comment (mwiktorin) [inline src/main/java/org/awesomebakery/agents/DoughPreparerAgent.java]: I would keep the old name here

File: src/main/java/org/awesomebakery/Start.java [hunk 6]
```diff
@@ -74,11 +74,11 @@ public static void main(String[] args) throws Throwable {
 		OvenManager ovenManager = new OvenManager("OvenManager");
 		mainContainer.acceptNewAgent("OvenManager", ovenManager).start();
 
-		DoughPreparer doughPreparer = new DoughPreparer("DoughPreparer");
-		mainContainer.acceptNewAgent("DoughPreparer", doughPreparer).start();
+		DoughPreparerAgent doughPreparerAgent = new DoughPreparerAgent("DoughPreparerAgent");
+		mainContainer.acceptNewAgent("DoughPreparerAgent", doughPreparerAgent).start();
```
Comment (mwiktorin) [inline src/main/java/org/awesomebakery/Start.java]: The String names of the agent should still be DoughPreparer i think

File: src/main/java/org/awesomebakery/Start.java [hunk 7]
```diff
@@ -74,11 +74,11 @@ public static void main(String[] args) throws Throwable {
...
 
-		DoughRester doughRester = new DoughRester("DoughRester");
-		mainContainer.acceptNewAgent("DoughRester", doughRester).start();
+		DoughResterAgent doughResterAgent = new DoughResterAgent("DoughResterAgent");
+		mainContainer.acceptNewAgent("DoughResterAgent", doughResterAgent).start();
```
Comment (mwiktorin) [inline src/main/java/org/awesomebakery/Start.java]: The Strings here should be without agent too

**Comment by sonarcube** [comment]
SonarQube analysis reported 6 issues
* ![INFO][INFO] 6 info

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [DoughPreparerAgent.java#L21](https://github.com/HBRS-MAAS/project-awesomebakery/blob/f70ddf7060771eacecb87a06715aa2bf2b010176/src/main/java/org/awesomebakery/agents/DoughPreparerAgent.java#L21): Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)
1. ![INFO][INFO] [DoughResterAgent.java#L21](https://github.com/HBRS-MAAS/project-awesomebakery/blob/f70ddf7060771eacecb87a06715aa2bf2b010176/src/main/java/org/awesomebakery/agents/DoughResterAgent.java#L21): Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)
1. ![INFO][INFO] [TruckAgent.java#L21](https://github.com/HBRS-MAAS/project-awesomebakery/blob/f70ddf7060771eacecb87a06715aa2bf2b010176/src/main/java/org/awesomebakery/agents/TruckAgent.java#L21): Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

**Pull request merged**